app/HTML.js

```diff
@@ -29,10 +30,17 @@ const renderers = {
         }}
       />);
     }
+    else if (htmlAttribs.class === "wp-block-embed__wrapper") {
```

Comment:
When exactly does `wp-block-embed__wrapper` show up? Could you add a comment?

Reply:
I added it for Spotify embeds, but I realized it also shows up for other things like Twitter embeds. I tried to make a custom render for iframes specifically to check the if the class has "Spotify" in it but I'm having trouble with making iframes show up.